src/main/java/equipments/technology/InternetComputer.java

```diff
@@ -0,0 +1,12 @@
+package equipments.technology;
+
+public class InternetComputer extends TechnologyEquipment {
+    public InternetComputer(String IP){
+        this.IP = IP;
+        this.technologyEquipmentType = "Internet Computer";
+    }
+    public String equipmentType() {
+        return "Internet Computer";
```

Comment:
Code dulpication- why is this defined both in `this.technologyEquipmentType` and in `equipmentType()`?